high-level-tests/e2e/cypress/integration/pix-app/a11y-authenticated-1.test.js

```diff
@@ -47,13 +49,12 @@ describe("a11y", () => {
         // when
         cy.visitMonPix(url);
         cy.get(".app-loader").should("not.exist");
-
         cy.injectAxe();
 
         // then
         viewports.forEach(({ width, height }) => {
           cy.viewport(width, height);
-          cy.checkA11yAndShowViolations({ skipFailures, url });
+          cy.checkA11yAndShowViolations({ skipFailures: false, url });
```

Comment:
`skipFailures` was set to `true` for some urls, notably `/assessments/fake-assessment`